packages/cms-lib/__tests__/ignoreRules.js

```diff
@@ -42,5 +42,9 @@ describe('ignoreRules', () => {
     it('does not ignore allowed files', () => {
       expect(shouldIgnoreFile(`${REPO_FOLDER}/hiddenFile.js`, CWD)).toBe(false);
     });
+
+    it('does not ignore the current folder', () => {
+      expect(shouldIgnoreFile(CWD, CWD)).toBe(false);
```

Comment:
Sorry for not being clear, but I was thinking something like `shouldIgnoreFile('.', CWD)` or `shouldIgnoreFile('', CWD)`. The expectation is the path passed to the function is a path to a file so it feels a little odd to have a test where `CWD` is passed to both.

Reply:
I checked it and when the bug occurs, the first argument equals the CWD. Consider these semantics of path.relative:
```
> path.relative('.','/a/b')
'../../../a/b'
> path.relative('','/a/b')
'../../../a/b'
> path.relative('/a/b','/a/b')
''
```

So you're suggestion wouldn't test what I fixed…

Reply:
yeah, in this case, it would need to be `process.cwd()` instead of the hard-coded CWD.

Reply:
```suggestion
      expect(shouldIgnoreFile('.', process.cwd())).toBe(false);
```

Reply:
Now I understand your suggestion. Great idea.